backend/app/services/chat_shell_model_service.py

```
@@ -73,14 +80,22 @@ def extract_response_text(response: Any) -> str:
+    logger.info(
+        f"[extract_response_text] from output array: {result[:100]}, texts count: {len(texts)}"
+    )
+    return result
 
 
 def _extract_text_from_sse_blob(raw: str) -> str:
     """Best-effort parser for SSE payload accidentally returned as plain text."""
     events: list[dict[str, Any]] = []
-    events.extend(_parse_sse_events_from_data_lines(raw))
-    events.extend(_parse_sse_events_from_blob(raw))
+    # 只使用一种解析方式，避免重复
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)